solr_configs/catalog-production-v2/conf/schema.xml

```diff
@@ -530,6 +530,13 @@
    <field name="subject_geo_facet" type="string" indexed="true" stored="true" multiValued="true" />
    <!-- pub_date is used for facet and display so it must be indexed and stored -->
    <field name="pub_date" type="string" indexed="true" stored="true" multiValued="true"/>
+   <field name="lc_subject_facet" type="string" indexed="true" stored="false" multiValued="true" />
```

Comment:
These look fine, but do we need them, since we have the `*_facet` dynamic field?

Reply:
@maxkadel yes, apparently we need them otherwise it can't search on them.